CourseDevelopment/Week8/Readme.MD

```diff
@@ -0,0 +1,194 @@
+# Course Development Week 8 The scientific process, beyond the PhD
+
+Course Development Week 8 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Feb/28/2023
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: open design and open hardware; prototyping? 
+
+Dalle2 & @hendelhendel: A grant proposal and money for science, while a duck is doing research in her lab.
+![eendjefunding](https://user-images.githubusercontent.com/93695286/222175974-54ffc694-d1b4-4e7b-bdd7-7f9d8c94f2d8.png)
+
+## Keywords student familiarity check week 8
+
+|**New Concepts**|**Familiar Concepts**|
+|----------------|---------------|
+|Careers in science and beyond | Mental health in academia|
+|Applying for scientific positions | Literature and referencing|
+|Data stewardship|CV and applications|
+|Grant writing|
+|the PhD advantage|
```

Comment:
This is maybe a bit of a confusing term. Advantage with respect to what?

Reply:
I read is as: the PhD advantage = The advantage of having a PhD in the rest of somebody's Career. 
So if one would like to stay in academia or move to R&D, a PhD is needed to be successful in such a career path. If a person is interested in e.g. business, teaching or politics, a PhD may not be necessary and can even be seen as an over qualification. This can hinder a career.

Reply:
OK, makes sense! Perhaps change then to 'The benefits of having a PhD'?